src/api/controller/api/characteristic.js

```diff
@@ -4,13 +4,25 @@ import route from 'koa-route';
 const app = new Koa();
 
 export const updateCharacteristics = async (ctx) => {
-    const characteristics = ctx.request.body;
-    const newCharacteristics = await Promise.all(characteristics.map(({
-        _id,
-        value,
-    }) => ctx.publishedCharacteristic.updateValueById(_id, value)));
+    const requestedNewCharacteristics = ctx.request.body;
+    const characteristics = await ctx.publishedCharacteristic.findLastVersion();
 
-    ctx.body = newCharacteristics.map(r => r.value);
+    const newCharacteristics = Object
+        .keys(characteristics)
+        .filter(key => key !== 'publicationDate')
+        .reduce((result, name) => {
+            const newCharacteristic = requestedNewCharacteristics[name];
+
+            return {
+                ...result,
+                [name]: {
+                    scheme: characteristics[name].scheme,
```

Comment:
Not sure we need to add the scheme inside the characteristics since we have the info in the fields.